front-end/src/app/cruds/pacientes/paciente-listar/paciente-listar.component.ts

```diff
@@ -38,12 +38,15 @@ export class PacienteListarComponent implements OnInit {
       icon: 'pi pi-info-circle',
 
       accept: () => {
-        this.pessoaService.deletaPaciente(id).subscribe(paciente => { }, err => { console.log('Erro ao deletar paciente') });
-        this.msgs = [{ severity: 'info', summary: 'Concluído', detail: 'Registro Excluido' }];
-        this.confirmationService.close();
-        location.reload;
+        this.pessoaService.deletaPaciente(id).subscribe(paciente => { this.msgs = [];
+          this.msgs = [{ severity: 'info', summary: 'Concluído', detail: 'Registro Excluido' }];
+          this.confirmationService.close();
+          location.reload; },
+           err => { console.log('Erro ao deletar paciente') });
+        
```

Comment:
Coloque o `this.confirmationService.close()` fora do subscribe, mas dentro do accept.